client/lib/posts/post-counts-store.js

```diff
@@ -19,30 +19,19 @@ const sum = obj => {
 var emitter = require( 'lib/mixins/emitter' ),
 	PostListStore = require( './post-list-store-factory' )(),
 	PostsStore = require( './posts-store' ),
-	sites = require( 'lib/sites-list' )(),
 	postUtils = require( 'lib/posts/utils' ),
 	Dispatcher = require( 'dispatcher' );
 
 var _counts = {},
 	PostCountsStore;
 
 /**
- * Get a normalized numberic siteId
- * @param {String|Integer} id - A site id (numeric site ID, site.slug, or API $site path fragment
- * @return {Integer} - normalized numeric site id
+ * Get siteId to use
+ * @param {Integer} id - A site id (numeric site ID)
```

Comment:
Have you looked up usages? Are we certain that all folks calling `getSiteId` are not passing through a site slug?